### PR TITLE
Just to add a few things

### PR DESCRIPTION
**Pull request by Dispersia**
As I put in the first commit: Updated GUI to be more user friendly / eye candy, added changeable background in settings. Fixed a few issues that could cause a crash / not able to queue.

### CODE REVIEW

File: LegendaryClient/Controls/GameSeperator.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+using System.Timers;
+using System.Windows.Media;
+using System.Windows.Threading;
+using LegendaryClient.Windows;
+using System.Text.RegularExpressions;
+
+namespace LegendaryClient.Controls
+{
+    /// <summary>
+    /// Interaction logic for JoinQueue.xaml
```
Comment (GodLesZ) [inline LegendaryClient/Controls/GameSeperator.xaml.cs]: Copy & paste mistake

File: LegendaryClient/Logic/Client.cs [hunk 2]
```diff
@@ -845,7 +847,7 @@ internal static string GetGameDirectory()
         }
 
 
-        private static int counter;
+        //private static int counter = 0;
```
Comment (GodLesZ) [inline LegendaryClient/Logic/Client.cs]: You don't have to comment things out - this is the big profit of a version control with history. 
You are able to revert everything at anytime.
It's 100% safe to remove things

File: LegendaryClient/Windows/PatcherPage.xaml.cs [hunk 3]
```diff
@@ -201,15 +202,14 @@ private void StartPatcher()
                     VersionAIR.Write(encoding.GetBytes("0.0.0.0"), 0, encoding.GetBytes("0.0.0.0").Length);
                     VersionAIR.Close();
                 }
-
                 string LatestAIR = patcher.GetLatestAir();
                 LogTextBox("Air Assets Version: " + LatestAIR);
                 string AirVersion = File.ReadAllText(Path.Combine(Client.ExecutingDirectory, "Assets", "VERSION_AIR"));
                 LogTextBox("Current Air Assets Version: " + AirVersion);
-                bool RetrieveCurrentInstallation = false;
+                //bool RetrieveCurrentInstallation = false; //just added this?
```
Comment (GodLesZ) [inline LegendaryClient/Windows/PatcherPage.xaml.cs]: Same again; it's safe to remove it if you dont need it

File: LegendaryClient/Windows/PatcherPage.xaml.cs [hunk 4]
```diff
@@ -328,7 +329,7 @@ private string GetLolRootPath()
                 return string.Empty;
         }
 
-        [Obsolete]
+        /*[Obsolete]
```
Comment (GodLesZ) [inline LegendaryClient/Windows/PatcherPage.xaml.cs]: Please remove

File: LegendaryClient/Windows/PlayPage.xaml.cs [hunk 5]
```diff
@@ -25,9 +25,11 @@ public partial class PlayPage : Page
         private int i = 0;
         private static Timer PingTimer;
         private Dictionary<double, JoinQueue> configs = new Dictionary<double, JoinQueue>();
+        private Dictionary<double, GameSeperator> gameConfigs = new Dictionary<double, GameSeperator>(); //Add functionality later
         private Dictionary<Button, int> ButtonTimers = new Dictionary<Button, int>();
+        private GameSeperator[] seperators = new GameSeperator[3];
         private List<double> Queues = new List<double>();
-        private int QueueID;
+        //private int QueueID;
```
Comment (GodLesZ) [inline LegendaryClient/Windows/PlayPage.xaml.cs]: Please remove

File: LegendaryClient/Windows/TeamBuilderPage.xaml.cs [hunk 6]
```diff
@@ -137,7 +137,7 @@ private void StartQueue(object sender, RoutedEventArgs e)
             string posUp = string.Format(position.ToUpper());
             string Json = string.Format("\"skinId\":{0},\"position\":\"{1}\",\"role\":\"{2}\",\"championId\":{3},\"spell2Id\":{4},\"queueId\":61,\"spell1Id\":{5}", skinId, posUp, roleUp, ChampionId, spell2, spell1);
             string JsonWithBrackets = "{" + Json + "}";
-            string TestJson = "{\"skinId\":0,\"position\"MIDDLE\",\"spell1:}";
+            //string TestJson = "{\"skinId\":0,\"position\"MIDDLE\",\"spell1:}";
```
Comment (GodLesZ) [inline LegendaryClient/Windows/TeamBuilderPage.xaml.cs]: Please remove

File: LegendaryClient/Windows/TeamQueuePage.xaml.cs [hunk 7]
```diff
@@ -41,7 +41,7 @@ namespace LegendaryClient.Windows
     /// </summary>
     public partial class TeamQueuePage : Page
     {
-        Message MessageData;
+        //Message MessageData;
```
Comment (GodLesZ) [inline LegendaryClient/Windows/TeamQueuePage.xaml.cs]: Please remove

**Comment by Dispersia** [comment]
Believe I reverted it - I couldn't figure out how to revert everything as you said so I could just put it back but hey. Again, sorry, all of these commands are confusing me lol (but thank you for the comments to attempt to get it fixed) can't for the life of me revert the teamqueuepage and replace it with Diamond I, but those aren't actual issues.

**Pull request merged**